management/ssl_certificates.py

```diff
@@ -249,6 +250,7 @@ def rdata__str__(r):
 				s = r.to_text()
 				if isinstance(s, bytes):
 					 s = s.decode('utf-8')
+				s = str(ipaddress.ip_address(s))
```

Comment:
You know the stuff better than I do, but without that line the IP would not be collapsed into the format given by str(ipaddress.ip_address(x)). From what I've seen, it is the same format the env["PUBLIC_IPV6"]) would be in. That being said, I think it would work if I just replaced all instances of env["PUBLIC_IPV6"]) with str(ipaddress.ip_address(env["PUBLIC_IPV6"]))). That would make it so whether you're checking against the v6 address from the queries or whether you're pulling the address from the ENV, it will be parsed through the ipaddress module and the format will always be the same. What do you think?